lib/msf/core/db_manager/host.rb

```diff
@@ -270,8 +270,12 @@ def report_host(opts)
   end
 
   def update_host(opts)
-    # TODO: remove unneeded workspace from opts until it's not automatically added to remote requests
-    opts.delete(:workspace)
+    # process workspace string for update if included in opts
+    wspace = opts.delete(:workspace)
```

Comment:
Just to confirm, we are going to only honor `:workspace` going forward and not support `wspace`?

Reply:
I think that's up for discussion, however, @clee-r7 mentioned being in the process of ridding the code of `:wspace`. I figured in the interim the above correction would be fine as the update method is new and the calling code is not including any `:wspace`.